src/reference/informative/informC02_units_reference.rst

```diff
@@ -8,4 +8,155 @@
 
   .. container:: infospec
 
-    **TODO**
\ No newline at end of file
+    .. container:: heading3
+
+      Understanding units references
+
+    Units are different from other element types in that their names could refer either to other units which you've imported or created in your model, or to one of the list of :ref:`built-in units<table_built_in_units>`.  
+    The same naming conventions apply here as elsewhere, so that part is fairly straightforward.
+
+    The trickier part is understanding the *scope* or *domain* in which named units exist, which is what points 1 and 2.2 are alluding to.
```

Comment:
Only if you're using imports (which most users aren't), otherwise it's very simple!
Maybe make sure to mention imports in this sentence? Something to say "it's easy unless you use imports, now I'm gonna talk about why" (so that readers who aren't importing can stop reading, or at least to set the scene!)